bin/systemml-spark-submit.py

```diff
@@ -0,0 +1,178 @@
+cparser.add_argument('--driver-memory', default='5G', help='Memory for driver (e.g. 512M)', metavar='')
+cparser.add_argument('--num-executors', default='2', help='Number of executors to launch', metavar='')
+cparser.add_argument('--executor-memory', default='2G', help='Memory per executor', metavar='')
+cparser.add_argument('--executor-cores', default='1', help='Number of cores', metavar='')
+cparser.add_argument('--conf', help='Spark configuration file', nargs='+', metavar='')
+
+# SYSTEM-ML Options
+cparser.add_argument('-nvargs', help='List of attributeName-attributeValue pairs', nargs='+', metavar='')
+cparser.add_argument('-args', help='List of positional argument values', metavar='', nargs='+')
+cparser.add_argument('-config', help='System-ML configuration file (e.g SystemML-config.xml)', metavar='')
+cparser.add_argument('-exec', default='hybrid_spark', help='System-ML backend (e.g spark, spark-hybrid)', metavar='')
+cparser.add_argument('-explain', help='explains plan levels can be hops, runtime, '
+                                      'recompile_hops, recompile_runtime', nargs='?', const='runtime', metavar='')
+cparser.add_argument('-debug', help='runs in debug mode', action='store_true')
+cparser.add_argument('-stats', help='Monitor and report caching/recompilation statistics, '
+                                    'heavy hitter <count> is 10 unless overridden', nargs='?', const='10', metavar='')
+cparser.add_argument('-gpu', help='uses CUDA instructions when reasonable, '
+                                  'set <force> option to skip conservative memory estimates '
+                                  'and use GPU wherever possible', action='store_true')
+cparser.add_argument('-f', required=True, help='specifies dml/pydml file to execute; '
+                                               'path can be local/hdfs/gpfs', metavar='')
+
+args = cparser.parse_args()
+
+# Optional arguments
+ml_options = []
+if args.nvargs is not None:
+    ml_options.append('-nvargs')
+    ml_options.append(' '.join(args.nvargs))
+if args.args is not None:
+    ml_options.append('-args')
+    ml_options.append(' '.join(args.args))
+if args.debug is not False:
+    ml_options.append('-debug')
+if args.explain is not None:
+    ml_options.append('-explain')
+    ml_options.append(args.explain)
+if args.gpu is not False:
+    ml_options.append('-gpu')
+if args.stats is not None:
+    ml_options.append('-stats')
+    ml_options.append(args.stats)
+
+# Assign script file to name received from argparse module
+script_file = args.f
+
+# find the systemML root path which contains the bin folder, the script folder and the target folder
+# tolerate path with spaces
+script_dir = os.path.dirname(os.path.realpath(__file__))
+project_root_dir = os.path.dirname(script_dir)
+user_dir = os.getcwd()
+
+scripts_dir = join(project_root_dir, 'scripts')
+build_dir = join(project_root_dir, 'target')
+target_jars = join(build_dir, '*.jar')
```

Comment:
You want to include 
${project_root_dir}/target/*.jar
and
${project_root_dir}/target/lib/*.jar